Core/Graphics/Include/UWP/CoreWindow/Babylon/GraphicsPlatform.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <windows.ui.core.h>
+#include <winrt/windows.ui.core.h>
+#include <winrt/Windows.ApplicationModel.h>
+
+using WindowType = winrt::Windows::UI::Core::ICoreWindow*;
+
+struct GraphicsConfiguration
+{
+    WindowType windowPtr;
+    void* windowTypePtr;
```

Comment:
You should be able to remove this as it is just supposed to indicate whether you are targeting an `ICoreWindow*` or an `ISwapChainPanel*`, which you already know at compile time.

Reply:
Yeah, I moved the logic to ConfigureBgfxPlatformData for the different platforms.